docs/products/redis/howto/enable-redis-ccr.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Enable cross-cluster replication in Aiven for Redis®*
+sidebar_label: Enable CCR
+pro: true
+---
+[Aiven for Redis cross-cluster replication (CCR)](/docs/products/redis/concepts/redis-ccr) feature enables data replication from a primary to a replica service, ensuring high availability and disaster recovery. Learn how to enable CCR for your Aiven for Redis service, verify data replication, and manage failovers.
+
+
+## Prerequisites
+
+- This feature requires `[Pro Platform](/docs/platform/concepts/pro-platform)`.
+- An Aiven account and API token
+- [Aiven CLI tool](https://github.com/aiven/aiven-client)
+- Two Aiven for Redis services within your Aiven project: one designated as the *primary*
+  and the other as the *replica*
+
+## Enable CCR via API
+
+To enable CCR for your Aiven for Redis service via API, follow these steps:
+
+1. **Create the primary service:** Start by creating your primary Aiven for Redis service.
+   This service acts as the primary (leader) in the replication setup. Use the
+   following Aiven API call to create the service:
+
+   ```shell
+      curl -X POST https://api.aiven.io/v1/project/PROJECT_NAME/service \
+      -H "Content-Type: application/json" \
+      -H "Authorization: bearer YOUR_AUTH_TOKEN" \
+      -d '{
+         "cloud": "CLOUD_NAME",
+         "plan": "business-4",
+         "service_name": "PRIMARY_SERVICE_NAME",
+         "service_type": "redis",
+         "project": "PROJECT_NAME"
+         }'
+   ```
+
+    Replace the placeholders with your actual values:
+
+    - `PROJECT_NAME`: The name of your project.
+    - `YOUR_AUTH_TOKEN`: Your API authentication token.
+    - `CLOUD_NAME`: The name of the cloud you want to use.
+    - `PRIMARY_SERVICE_NAME`: The name you want to give to your primary service.
+    - `YOUR_AUTH_TOKEN`: Your API authentication token.
+
+1. **Confirm primary service backups:** Before creating a replica service, verify that
+   your primary Aiven for Redis service has a backup.
+   [Configuring backups](/docs/platform/concepts/service_backups#aiven-for-redis)
+   is essential to ensure data consistency and integrity for replication.
+
+1. **Create the replica service:** Next, create the Aiven for Redis service that acts as
+   the replica. Specify the primary service as the source for replication using the
+   following API call:
+
+   ```shell
+    curl -X POST https://api.aiven.io/v1/project/PROJECT_NAME/service \
```

Comment:
commands looks good but didn't run them myself now. I assume you did and they work?

Reply:
I didn't either; I based it on the examples in other services (PG, for example).